### PR TITLE
Create Handbook section

### PR DESCRIPTION
**Pull request by justinegeffen**

### CODE REVIEW

File: operations/operations/handbook/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Handbook-first means that when you have a new idea, a new process, an update to a process, a change in your team, etc you document it in the Handbook first. For example, when a new staff member joins us, they should be added to your team's page before their first day. Another example is if you've created a new Jira process - first document it in the Handbook and then announce it to your team/the company. This allows you to find blindspots and also means that people can go to the relevant page immediately after the announcement to read more.
+
+### How do we drive Handbook-first?
+
+Great question. It's a team effort across the whole company, but we have a secret weapon in the form of Handbook Champions.
+
+## Handbook Champions
+
+A Handbook Champion is a volunteer from a specific team/operational area who drives Handbook-first adoption and acts as a go-to in the team for Handbook questions. While they aren't responsible for a set num
```
Comment (cwarnermm) [inline operations/operations/handbook/README.md]: this sentence appears to need an ending :)

File: operations/operations/handbook/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### What do I need to do?
+
+To put it simply, anytime someone in your team/operational area mentions a process in a meeting or asks about one, remind them that they should either document it or can find it in the Handbook. You may also be asked to review PRs before they're merged. In time, you'll find that you're reminding less, and being asked to review more... until Handbook-first becomes a company habit.
+
+Of course, if you have ideas for content that's needed or content that should be updated and you're happy to tackle some of it that's fantastic. Likewise if you have suggestions, ideas, complaints, or issues - feel free to raise them.
+
+### How do I get involved?
+
+You can join the Handbook Champions channel, and then also update this list with your name and team/organizational area. There can be multiple champions per team.
```
Comment (cwarnermm) [inline operations/operations/handbook/README.md]: Consider adding a link to the channel

File: operations/operations/handbook/handbook-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+The Mattermost Handbook is a work-in-progress in that information is constantly being updated, added, revised, and edited.
+
+## Editing the Handbook
+
+The Handbook is open to the public and edits can be submitted by anyone.
+
+## Adding content
+
+You don't need to ask permission to add content to the Handbook. Follow the steps in "How do I edit the Handbook" to get started.
```
Comment (cwarnermm) [inline operations/operations/handbook/handbook-onboarding.md]: A direct link here would be ideal

**Review by cwarnermm — DISMISSED** [review]
Non-blocking feedback provided inline and 1 sentence fragment detected :)

File: operations/operations/handbook/README.md (hunk 1 above)
Comment (justinegeffen) [reply operations/operations/handbook/README.md]: That definitely seems like the case. :D

File: operations/operations/handbook/handbook-onboarding.md (hunk 3 above)
Comment (justinegeffen) [inline operations/operations/handbook/handbook-onboarding.md]:
```suggestion
You don't need to ask permission to add content to the Handbook. Follow the steps in [How to update the Handbook](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook) to get started.
```

**Review by cwarnermm — APPROVED** [review]
This is a fantastic update!! Thank you!

**Pull request merged**